scripts/azure_logs_integration_original_DD.js

```diff
@@ -646,12 +669,6 @@ class EventhubLogHandler {
 }
 
 module.exports = async function (context, eventHubMessages) {
-  if (!DD_API_KEY || DD_API_KEY === "<DATADOG_API_KEY>") {
```

Comment:
At some point folks will ask for key support; might as well preserve this.

Reply:
do you know what would be the right header for the api key for us? Would it be `Kf_Api_Key`? (cc @vinayk-kfuse )

Reply:
After discussing with @harold-kfuse, added "KF_API_KEY_ENV" in the header for authentication.

Reply:
changed it to DD_API_KEY